models/action.go

```diff
@@ -136,6 +137,12 @@ func (a Action) GetIssueInfos() []string {
 	return strings.SplitN(a.Content, "|", 2)
 }
 
+func (a Action) GetIssueTitle() string {
+	issueID := com.StrTo(a.GetIssueInfos()[0]).MustInt64()
+	issue, _ := GetIssueByID(issueID)
```

Comment:
Should we check error here?

`if err !=nil` then `log.Error(4, "GetIssueByID: %v", err)` and returns somethings like `500 when get title` as title?